nncf/openvino/engine.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, model: ov.Model, target_device: TargetDevice = TargetDevice.C
 
         ie = ov.Core()
         stateful = model_has_state(model)
-        compiled_model = ie.compile_model(model, target_device.value)
+        compiled_model = ie.compile_model(model, device_name="CPU")
```

Comment:
I would say this is an inconsistent change. Does nncf use `OVNativeEngine` with target_device?

Reply:
Removed. Thank you.